### PR TITLE
Create Readme.md

### PR DESCRIPTION
**Pull request by psanjay-1**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<h1>Atm-near -me is an website which allows the users to find the cash machines around them.<h1>
```
Comment (Ritsyy) [inline Readme.md]: You gave an extra space here, it should be `Atm-near-me`

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<h2> It is an open source webiste so any one can contribute to the project and send new ideas. <h3>
```
Comment (Ritsyy) [inline Readme.md]: typo for `website`

File: Readme.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<h1>Atm-near -me is an website which allows the users to find the cash machines around them.<h1>
+<h2> It is an open source webiste so any one can contribute to the project and send new ideas. <h3> 
+<h3>The next challanges for the project are:
```
Comment (Ritsyy) [inline Readme.md]: it should be `challenges`

**Review by Ritsyy — COMMENTED** [review]
Thank you for this PR, very much appreciated!

File: Readme.md (hunk 1 above)
Comment (psanjay-1) [reply Readme.md]: Done

File: Readme.md (hunk 2 above)
Comment (psanjay-1) [reply Readme.md]: Done

File: Readme.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<h1>Atm-near-me is an website which allows the users to find the cash machines around them.<h1>
+<h2> It is an open source website so any one can contribute to the project and send new ideas. <h3> 
+<h3>The next challenges for the project are:
+<p>1) The user should be able to select certain radius around him so the website will only look for the machines in theat specific distance.(kms or miles).
```
Comment (Ritsyy) [inline Readme.md]: It will be `that specific distance`.

**Pull request merged**